developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -1270,7 +1273,7 @@ KMX_BOOL CheckContextStatementPositions(PKMX_WCHAR context) {
 /**
  *  Checks if a use() statement is followed by other content in the output of a rule
  */
-KMX_DWORD CheckUseStatementsInOutput(const PFILE_GROUP gp,PKMX_WCHAR output) {
+KMX_DWORD CheckUseStatementsInOutput(const PFILE_GROUP /*gp*/,PKMX_WCHAR output) {
```

Comment:
Let's fix the declaration and use if `gp` is not used:

```suggestion
KMX_DWORD CheckUseStatementsInOutput(PKMX_WCHAR output) {
```

Reply:
+1.  Wasn't sure if it was meant to be a parallel to something else, etc.

Reply:
done